app/views/merge_requests/absorbing_organisation.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :before_content do %>
+  <% title = "Tell us if your organisation is merging" %>
+  <% content_for :title, title %>
+  <%= govuk_back_link(href: :back) %>
```

Comment:
Do we want this back button to take us back in the form flow? For example, currently if we're on `/merge-request/:id/confirm-telephone-number` and keep on pressing back, we get stuck in a loop